src/components/Board/PagenationBoard/index.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Created on Wed Nov 15 2023
+ *
+ * Copyright (c) 2023 Your Company
+ */
+
+import { Button, Grid } from "@/components";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+
+export default function PagenationBoard() {
+  interface PostData {
+    author: string;
+    date: string; //추후 api 스펙 관련 필요시 변경
+    title: string;
+    view: number;
+    postNum: number;
+  }
+
+  const dummyPage: number[] = [1, 2, 3];
+  const dummyData: PostData[] = [
+    { author: "pho", date: "23.11.15", title: "ㅎㅇ", view: 33, postNum: 0 },
+    { author: "jbh", date: "23.11.13", title: "ㅎㅇgd", view: 11, postNum: 0 },
+    {
+      author: "skm",
+      date: "23.11.12",
+      title: "ㅎㅇㅎㅇ",
+      view: 22,
+      postNum: 0,
+    },
+  ];
+
+  const router = useRouter();
+
+  function onClickButton(ev: React.MouseEvent<HTMLButtonElement>) {
+    router.push({
+      query: { page: ev.target.innerHTML },
+    });
+  }
+
+  function onClickPost() {
+    router.push({
+      //
+    });
+  }
```

Comment:
```suggestion
  function handleClickButton(ev: React.MouseEvent<HTMLButtonElement>) {
    router.push({
      query: { page: ev.target.innerHTML },
    });
  }

  function handleClickPost() {
    router.push({
      //
    });
  }
```

on이라는 접두사는 일반적으로 이벤트를 받는쪽에서 사용하고 <br/>
이벤트를 선언하고 전달하는 쪽에서는 일반적으로 handle이라는 접두사를 사용합니다.

Reply:
반영했습니다